packages/blade/src/components/BottomSheet/__tests__/BottomSheet.ssr.test.tsx

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable @typescript-eslint/no-empty-function */
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+import { BottomSheet, BottomSheetBody, BottomSheetFooter, BottomSheetHeader } from '../BottomSheet';
+import renderWithSSR from '~utils/testing/renderWithSSR.web';
+import { Text } from '~components/Typography';
+import { Badge } from '~components/Badge';
+import { Counter } from '~components/Counter';
+import { Button } from '~components/Button';
+
+beforeAll(() => jest.spyOn(console, 'error').mockImplementation());
```

Comment:
Why?Is there any error coming up specifically we want to suppress?

Reply:
Act warnings are coming because there are some async logic inside FloatingUI.